src/crawlee/crawlers/_adaptive_playwright/_adaptive_playwright_crawler.py

```diff
@@ -25,6 +25,7 @@
     PlaywrightCrawlingContext,
     PlaywrightPreNavCrawlingContext,
 )
+from crawlee.crawlers._basic._basic_crawler import _BasicCrawlerOptions
```

Comment:
I'm sorry, but could you please rename `_BasicCrawlerOptions` to `BasicCrawlerOptions` since we're importing it from another module?